ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml

```diff
@@ -850,6 +862,18 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi
+- as: azure-ipi-amd-f7-netobserv
```

Comment:
per the alphabet order, I think the job should be placed prior to `azure-ipi-amd-mixarch-f9-destructive`, right?